modules/sdk-coin-atom/src/lib/utils.ts

```diff
@@ -39,9 +39,15 @@ export class Utils implements BaseUtils {
 
   /** @inheritdoc */
   isValidTransactionId(txId: string): boolean {
-    throw new Error('Method not implemented.');
+    return this.validateBlake2b(txId);
   }
 
+  validateBlake2b(hash: string): boolean {
```

Comment:
this function can be static. We won't require redundant `this.`  Also, blake2 and sha256 hash has the same size of hash, not sure if we need separate function

Reply:
+1. Also, seems like we're just checking length along with regex character matching. Any thoughts on why the method is named this way ?